hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieCommonTestHarness.java

```diff
@@ -91,6 +91,12 @@ protected void initMetaClient() throws IOException {
     basePath = metaClient.getBasePath();
   }
 
+  protected void cleanMetaClient() {
```

Comment:
you could actually register this with `@AfterEach` ? so that subclasses won't have to call this

Reply:
> you could actually register this with `@AfterEach` ? so that subclasses won't have to call this

There is no `set up` method provided in HoodieCommonTestHarness, the subclass controls the objects that need to be initialized, so the subclass controls the cleanup as well.